static/app/components/events/interfaces/crashContent/exception/content.tsx

```diff
@@ -35,15 +40,32 @@ export function Content({
   groupingCurrentLevel,
   hasHierarchicalGrouping,
   platform,
+  projectSlug,
   values,
   type,
   meta,
 }: Props) {
+  // Organization context may be unavailable for the shared event view, so we
+  // avoid using the `useOrganization` hook here and directly useContext
+  // instead.
+  const organization = useContext(OrganizationContext);
   if (!values) {
     return null;
   }
 
+  const orgHasSourcemapCta = organization?.features?.includes('fix-source-map-cta');
```

Comment:
@scttcper Make sure to disable the CTA that Priscilla added when this feature flag is true, this CTA is going to replace that

Reply:
yeah i check for that on line 88 i think

Reply:
yes please 🙏 we will have to remove the old feature flag as well